### ISSUE TITLE
set LC_TIME=C systemwide

### ISSUE THREAD

**Issue by philhagen** [teamdfir/sift#511]
set LC_TIME=C so all (most?) shell time utilities respect formatting

**Comment by philhagen** [comment]
would still like this to get in if not already

**Comment by ekristen** [comment]
Definitely. Added labels so it won't go stale again. I don't recall if this made it in the latest. I'll verify
